web-ng/src/app/components/layer-list/layer-list.component.ts

```diff
@@ -44,6 +48,16 @@ export class LayerListComponent {
           List(project.layers.valueSeq().toArray()).sortBy(l => l.index)
         )
       );
+    this.initLayerListPermission();
+  }
+
+  async initLayerListPermission(): Promise<void> {
```

Comment:
Since layer list component can never be shown without a project being loaded, can we do what we did with `currentUser` and expose `getCurrentProject()` from the `ProjectService`? Then this logic can all be synchronous. We `take(1)` here, so the current impl isn't reactive to remote changes to Project anyway.

For the edge case where permissions are changed while the user has the page open: We enforce ACLs server side, so when if they are in a UI which they have now been denied access to, we can show an error when the save will fails [TBD later].

Does that make sense?